vars/all_var.yml

```diff
@@ -273,97 +111,21 @@
   app_fhir_key_name: "{{ env_app_fhir_key_name }}"
 
-  # cf_app_selinux_policy sets SELINUXTYPE=targeted
-  cf_app_selinux_policy: "{{ selinux_policy }}"
-  # cf_app_selinux_state set SELINUX=permissive
-  cf_app_selinux_state: "{{ selinux_state }}"
-
-  cf_app_sg_app_zone_id: "{{ env_cf_app_sg_app_zone_id }}"
-  cf_app_azone: "{{env_cf_app_azone }}"
-  cf_app_build_image_id: "{{ env_cf_app_build_image_id }}"
-  cf_app_instance_name: "{{ env_cf_app_instance_name }}"
-  cf_app_tag_key_stack: "{{ env_cf_app_tag_key_stack }}"
+  # legacy values still used in roles, should be renamed for clarity
   cf_app_tag_key_layer: "{{ env_cf_app_tag_key_layer }}"
-  cf_app_key_name: "{{ env_cf_app_key_name }}"
-  cf_app_instance_type: "{{ env_cf_app_instance_type }}"
-  cf_app_public_server_name: "{{ env_cf_app_public_server_name }}"
-  cf_app_pyserver_name: "{{ env_cf_app_pyserver_name }}"
-  cf_app_pyapp_settings: "{{ env_cf_app_pyapp_settings }}"
-  cf_app_repo_branch: "{{ env_cf_app_repo_branch }}"
-  cf_app_db_user: "{{ env_cf_app_db_user }}"
-  cf_app_db_pwd: "{{ env_cf_app_db_pwd }}"
-  cf_app_pyapps_email: "{{ env_cf_app_pyapps_email }}"
   cf_app_pyapps_pwd: "{{ env_cf_app_pyapps_pwd }}"
-  # Removing Herd Member designation
-  # cf_app_herd_member: "{{ env_cf_app_herd_member }}"
-  cf_app_db_type: "{{ env_cf_app_db_type }}"
-  cf_app_db_prefix: "{{ env_cf_app_db_prefix }}"
-  cf_app_db_host_address: "{{ env_cf_db_host_address }}"
-  cf_app_target_vpc_id: "{{ cf_vpc_id }}"
-
-  # Postgres Role related settings
-  db_server_account: "postgres"
-  db_server_data_dir: "/data/pgsql/9.6/data"
-  db_server_temp_dir: "/data/pgbackup"
-  db_server_table_name: "{{ cf_data_database_prefix }}{{ cf_data_tag_key_stack }}_{{ common_db_type[cf_data_database_type]['DbNameTable'] }}"
-  db_server_backup_pub_key: "backup_key.pem.pub"
-  # role_ db_user: "postgres"
-  # role_db_data_dir: "/data/pgsql/9.6/data"
-  # role_db_temp_dir: "/data/pgbackup"
-  # role_db_table_name: "{{ cf_data_database_prefix }}{{ cf_data_tag_key_stack }}_{{ common_db_type[cf_data_database_type]['DbNameTable'] }}"
-  # role_db_backup_pub_key: "backup_key.pem.pub"
-
-  # CA Certificate Root password
-  ca_rootca_password: "{{ env_ca_rootca_password }}"
 
   mon_nessus_user: "{{ env_mon_nessus_user }}"
   mon_nessus_pwd: "{{ env_mon_nessus_pwd }}"
   mon_nessus_pub_key: "{{ env_mon_nessus_pub_key }}"
 
-  # Type of Elastic Load Balancer: app | net | classic | appclassic
-  elastic_load_balancer_type: "{{ env_elastic_load_balancer_type }}"
-
-  # Classic Load Balancer Name:
-  # follows format: BB-{env}-CLB-n
-  app_clb_name: "{{ env_app_clb_name }}"
-
-  app_lb_arn: "{{ env_app_lb_arn}}"
-  app_ssl_lb_arn: "{{ env_app_ssl_lb_arn }}"
-  app_lb_app_protocol: "https"
-
   # splunk configuration
   # Group name = BBAPI
   splunk_group_name: "{{ env_splunk_group_name }}"
   splunk_target_layer: "{{ env_splunk_target_layer }}"
-  splunk_client_name: "{{ env_splunk_client_name }}"
```

Comment:
The splunk client name is constructed from all other values directly in the splunk playbooks/roles so this value is never used from here.